src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authentication/AuthenticationConfig.kt

```diff
@@ -0,0 +1,55 @@
+package org.vaccineimpact.orderlyweb.security.authentication
+
+import org.pac4j.core.client.IndirectClient
+import org.pac4j.core.profile.CommonProfile
+import org.pac4j.core.credentials.Credentials
+import org.vaccineimpact.orderlyweb.db.AppConfig
+import org.vaccineimpact.orderlyweb.db.OrderlyUserRepository
+import org.vaccineimpact.orderlyweb.security.clients.MontaguIndirectClient
+import org.vaccineimpact.orderlyweb.security.clients.GithubIndirectClient
+
+class AuthenticationConfig
+{
+    companion object {
+        fun getConfiguredProvider(): AuthenticationProvider {
+            var configuredValue = AppConfig()["auth.provider"]
+            return when (configuredValue.toLowerCase()) {
+                "github" -> AuthenticationProvider.Github
+                "montagu" -> AuthenticationProvider.Montagu
+                "none" -> AuthenticationProvider.None // For testing purposes, or may be instances which don't require any auth
```

Comment:
Surreptitious introduction of 'none' authentication type. Do we want to support this?